models/amazon_selling_partner__orders.sql

```diff
@@ -0,0 +1,114 @@
+{{ config(enabled=var('amazon_selling_partner__using_orders_module', true)) }}
+
+with orders as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__orders')}}
+),
+
+order_item as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__order_item') }}
+),
+
+order_item_promotion_id as (
+    
+    select *
+    from {{ ref('stg_amazon_selling_partner__order_item_promotion_id') }}
+),
+
+payment_method_detail_item as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__payment_method_detail_item') }}
+),
+
+aggregate_order_items as (
+
+    select 
+        source_relation,
+        amazon_order_id,
+        count(order_item_id) as count_order_items,
+        sum(coalesce(item_price_amount, 0)) as total_item_price_amount,
+        {# Highly unlikely that a single order will use multiple currencies, but just to be safe  #}
```

Comment:
Makes sense! You can remove this comment

Reply:
removed